pulsar-broker/pom.xml

```diff
@@ -417,6 +417,12 @@
       <artifactId>javax.activation</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>io.rest-assured</groupId>
+      <artifactId>rest-assured</artifactId>
+      <scope>test</scope>
+    </dependency>
+
```

Comment:
I see the origin patch has another fix at line 29.

Reply:
It's not required and I think we also need to change the master branch to make it consistent as other components.

Reply:
btw. I `../pom.xml` is the default. We should be removing the outdated `<relativePath>..</relativePath>` entries completely. 
there's `<xs:element minOccurs="0" name="relativePath" type="xs:string" default="../pom.xml">` in the `maven-4.0.0.xsd` which shows the default.

Reply:
created https://github.com/apache/pulsar/pull/22741 to cover the pom.xml files